src/net.cpp

```diff
@@ -1223,41 +1223,40 @@ void CConnman::DisconnectNodes() {
 }
 
 void CConnman::InactivityChecks() {
-    LOCK(cs_vNodes);
-    for (CNode* pnode : vNodes) {
-        InactivityCheck(pnode);
-    }
-}
-
-void CConnman::InactivityCheck(CNode *pnode) {
     int64_t nMicroTime = GetTimeMicros();
```

Comment:
Commit "Merge InactivityCheck into InactivityChecks."

Should be after `LOCK(cs_vNodes)`?

Reply:
Not sure what you mean here

Reply:
Move the call `GetTimeMicros` after the lock?

Reply:
Ah I see